### ISSUE TITLE
ReferenceError: Zone is not defined 

### ISSUE THREAD

**Issue by blackecho** [driftyco/ionic2#800]
After I updated to alpha-45 from alpha-42, my app is not working. I have this error in ionic serve:

    ReferenceError: Zone is not defined

ionic info:

    Your system information:

    Cordova CLI: 5.4.1
    Gulp version:  CLI version 3.9.0
    Gulp local:   Local version 3.9.0
    Ionic Version: 2.0.0-alpha.45
    Ionic CLI Version: 2.0.0-beta.11
    Ionic App Lib Version: 2.0.0-beta.5
    OS: Distributor ID:	Ubuntu Description:	Ubuntu 15.10 
    Node Version: v4.0.0

project dependencies in package.json:

    "devDependencies": {
        "autoprefixer-loader": "^3.1.0",
        "awesome-typescript-loader": "^0.15.9",
        "css-loader": "^0.23.0",
        "del": "^2.2.0",
        "file-loader": "^0.8.4",
        "gulp": "^3.9.0",
        "gulp-autoprefixer": "^3.1.0",
        "gulp-sass": "^2.1.0",
        "gulp-watch": "^4.3.5",
        "node-sass": "^3.4.2",
        "resolve-url-loader": "^1.4.2",
        "sass-loader": "^3.1.1",
        "style-loader": "^0.13.0",
        "typescript": "^1.7.3",
        "url-loader": "^0.5.6",
        "webpack": "^1.12.4"
    },
    "dependencies": {
        "angular2": "2.0.0-beta.0",
        "es6-promise": "3.0.2",
        "es6-shim": "0.33.3",
        "ionic-framework": "2.0.0-alpha.45",
        "ionic-native": "~1.0.8",
        "reflect-metadata": "0.1.2",
        "rxjs": "5.0.0-beta.0",
        "zone.js": "0.5.10"
    },

**Comment by blackecho** [comment]
[SOLVED] I have updated the project structure and config file to reflect the changes in the conference app and everything is working now.

**Comment by jamespacileo** [comment]
Hey @blackecho ! I'm having this problem now... could you kindly share what changes you've made?
Not sure what changes I need to apply.

Pretty please... :smile:

**Comment by blackecho** [comment]
@jamespacileo I changed my project structure to be the same as the conference app.
https://github.com/driftyco/ionic-conference-app

Moved app folder outside www and modified all the imports and scripts accordingly.

Updated the config files:
    ionic.config.js
    package.json
    webpack.config.js

removed gulpfile.js, ran npm update to update the dependencies, and:

    sudo npm install -g ionic@beta

to update to the latest ionic cli.
